samples/04_gis_analysts_data_scientists/Training a Deep Learning model, for using it in 3D basemap Solutions..ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>One of the most important initial steps in the workflow of '3D Basemap' solutions, is the classification of LiDAR Point Clouds. This sample notebook provides a reference for how to train a deep learning model for point cloud classification. </p><p> </p><p>To achieve this, the ArcGIS API for Python's implementation of PointCNN is used. Further details on the PointCNN implementation in the API (working principle, architecture, best practices, etc.), can be found <a href="https://developers.arcgis.com/python/guide/point-cloud-segmentation-using-pointcnn" target="_blank">here</a>, along with instructions on how to set up the Python environment. Additional sample notebooks related to PointCNN can be found <a href="https://developers.arcgis.com/python/sample-notebooks/" target="_blank">here</a>.</p><p> </p><p>Before proceeding through this notebook, it is advised to go through the API Reference for PointCNN [<code>export_point_dataset( ), prepare_data( ) Transform3d( ), &amp; PointCNN( )</code>], that can be found <a href="http://developers.arcgis.com/python/api-reference/arcgis.learn.html" target="_blank">here</a>. It will help in understanding the PointCNN's workflow in detail. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
'corrected accordingly'', Thanks.